spec/bibliothecary_spec.rb

```diff
@@ -171,6 +171,112 @@
     Bibliothecary.reset
   end
 
+  it 'handles a complicated folder with many manifests complaining about no-parser files', :vcr do
+    # If we run the analysis in pwd, confusion about absolute vs.
+    # relative paths is concealed because both work
+    orig_pwd = Dir.pwd
+    analysis = Dir.chdir("/") do
+      described_class.analyse(File.join(orig_pwd, 'spec/fixtures/multimanifest_dir'), ignore_unparseable_files=false)
```

Comment:
oops I think this is actually python kwargs syntax and means something goofy in ruby